app/lib/customer.ts

```diff
@@ -0,0 +1,59 @@
+import { client } from '@/app/lib/ClientBuilder';
+import { FormInputs } from '@/app/registration/constants';
+
+const generateId = () => {
+  return Math.random().toString(36).substring(2, 11);
+};
```

Comment:
Common practice to use some UUID generator for this

Reply:
imported `v4` function from the `uuid` library to generate unique identifiers, rather than creating a fixed-length character set using custom JavaScript functions.